src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx

```diff
@@ -85,38 +83,12 @@ const FeatureFlagsPage: React.FC = () => {
     queryParams
   })
 
-  const [featureMetrics, setFeatureMetrics] = useState<FeatureMetric[]>()
-
-  useEffect(() => {
-    const loadMetrics = async (): Promise<void> => {
-      if (enableMetricsEndpoint) {
-        const metrics = await getFeatureMetricsPromise({
-          queryParams: {
-            accountIdentifier,
-            environmentIdentifier,
-            orgIdentifier,
-            projectIdentifier,
-            pageSize: CF_DEFAULT_PAGE_SIZE,
-            pageNumber,
-            [flagFilter.queryProps?.key]: flagFilter.queryProps?.value
-          }
-        })
-
-        setFeatureMetrics(metrics as FeatureMetric[])
-      }
+  const { data: featureMetrics, loading: featureMetricsLoading } = useGetFeatureMetrics({
+    queryParams: {
+      ...queryParams,
+      environmentIdentifier: environmentIdentifier
```

Comment:
Deffo not a blocker, but you can simplify
```suggestion
      environmentIdentifier
```